### ISSUE TITLE
Run panel code under process monitor

### ISSUE THREAD

**Issue by wearhere** [wearhere/spacecontrol#35]
So it'll restart on crash.

(General note that the server code is [run under a monitor](https://github.com/wearhere/spacecontrol/blob/b8356f7/gulpfile.js#L149) already.)

**Comment by wearhere** [comment]
@igor47 assigning to you since I recall you saying you had some idea here.

**Comment by igor47** [comment]
yeah, i was going to write a systemd control file. i'll work on it tomorrow
